File: project 3 - Array/python/Random_Array_Creator.py
import random

myArrays = []
num_random_numbers = 10

while len(myArrays) < num_random_numbers:
    random_number = random.randint(1, 100)
    
    if random_number in myArrays:
        print(f"{random_number} already exists in the array.")
    else:
        myArrays.append(random_number)
        print(f"Added {random_number} to the array.")

print("Final array of random numbers:", myArrays)

File: project 3 - Array/php/Random_Array_Creator.php
<?php
$myArrays = [];

if (isset($_GET['user_char'])) {
    $user_char = $_GET['user_char'];
    
    if (in_array($user_char, $myArrays)) {
        echo "$user_char is already exist.<br>";
        echo "Enter other words.<br>";
    } else {
        $myArrays[] = $user_char;
        echo "$user_char has been added.<br>";
    }
}

if (!empty($myArrays)) {
    echo "Current words: " . implode(", ", $myArrays) . "<br>";
}
?>

<form method="get">
    <input type="text" name="user_char" required>
    <input type="submit" value="Submit">
</form>